manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -506,11 +506,14 @@ properties:
       auctioneer:
         api_url: http://auctioneer.service.cf.internal:9016
       etcd:
-        machines: [(( grab properties.etcd.advertise_urls_dns_suffix ))]
-        ca_cert: (( grab meta.secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        machines: []
+        ca_cert: ""
```

Comment:
I think it's fine as-is, personally. We shouldn't get too hung up on it.

Presumably it will be removed at some point in the future when etcd support goes from bbs.